packages/authenticator/amplify_authenticator/lib/src/widgets/social/social_button.dart

```diff
@@ -205,24 +217,32 @@ class _SocialSignInButtonState
           builder: (context, constraints) {
             final padding = calculatePadding(constraints);
             return Padding(
-              padding: EdgeInsets.symmetric(horizontal: padding),
+              padding: EdgeInsets.symmetric(
+                vertical: 2,
```

Comment:
I think this is increasing the height of the button. Is that what we want? It seems like the social buttons have a larger height than other buttons here: https://github.com/aws-amplify/amplify-flutter/pull/3692/files#diff-19731debe5953de4ce3baae91c6a663bf423a41677f989d42e1448b480a54353